options/comments.md

```diff
@@ -93,8 +93,27 @@ Prevents warnings due to unspecified tags from being reported in comments within
 }
 ```
 
-Override TypeDoc's supported block tags, emit warnings for any tags not listed here.
-This option will be set by `tsdoc.json` if present.
+This specifics all of the [JSDoc block tags](../guides/tags.md#block-tags) that TypeDoc considers to be valid.
+
+By default, TypeDoc will warn when it finds a non-valid JSDoc tag. If you need to add a custom one, you can extend the defaults by using a JavaScript configuration file:
```

Comment:
```suggestion
By default, TypeDoc will warn when it finds an unknown tag. If you need to add a custom one, you can extend the defaults by using a JavaScript configuration file:
```

Saying non-valid or invalid implies the user is doing something wrong, which isn't necessarily the case, TypeDoc isn't necessarily the only tool parsing these comments